src/test/java/seedu/mindmymoney/command/AddCommandTest.java

```diff
@@ -0,0 +1,38 @@
+package seedu.mindmymoney.command;
```

Comment:
I think there is a folder to write all the JUnit tests. src > test > java > seedu.mindmymoney > MindMyMoneyTest. Maybe can add there instead? Or create a command folder similar to the one in main and add the AddCommandTest there.

Reply:
This one is in the command folder, its when i follow lecture video and right click generate test function
![image](https://user-images.githubusercontent.com/69450377/157637533-3751f468-beca-4399-a857-71eb45922311.png)

Reply:
Ahh I've missed that, my bad. Yup, that everything looks good!